dev-python/pymacaroons/files/pymacaroons-0.13.0-nose-to-pytest.patch

```diff
@@ -0,0 +1,445 @@
+This patch is backported from upstream PR #59 but only parts related to
```

Comment:
This file is too large for FILESDIR. Couldn't you fetch the `.patch` from upstream PR? Alternatively, perhaps we could just make a snapshot ebuild.

Reply:
I cannot fetch the `.patch` from upstream because it does not apply cleanly, it was necesary to modify it. However, yes, it would be probably possible to do a snapshot ebuild from current `master` and than fetch the patch from PR. Sounds a little bit weird :) Or I can do a snapshot from PR directly, this sounds even weirder :) What do you think?

Reply:
Hmm, or perhaps you could just fork the repo, add a commit on top of the tag and use that?

Reply:
(as a patch)

Reply:
ok, sounds good to me, I will do that.

Reply:
thanks

Reply:
done